discord/ext/test/runner.py

```diff
@@ -570,6 +570,8 @@ async def on_command_error(ctx, error):
             user = back.make_user(f"TestUser{str(num)}", f"{num+1:04}")
             member = back.make_member(user, guild, nick=user.name + f"_{str(num)}_nick")
             members.append(member)
+        client_member = back.make_member(client.user, guild, nick=client.user.name + f"_nick")
```

Comment:
I think this may actually be redundant with the bellow `back.make_member(back.get_state().user)`, maybe instead try just appending the result of that to members, instead of making it twice